frontend/src/hooks/useResources.ts

```diff
@@ -1,24 +1,76 @@
-import { useEffect, useState } from 'react';
 import i18n from 'i18next';
 
 import { ResourcesApi } from '@/api';
-import { ResourceModel } from '@/models/ResourceModel';
+import { useTranslation } from 'react-i18next';
+import { useEffect, useState } from 'react';
+import { LocalStorageService } from '@/services';
+
+type TLanguages = 'EN' | 'UA';
+
+const LANGUAGES: { [key in TLanguages]: string; } = {
```

Comment:
There is an API endpoint that can return the list supported of languages: is it possible to use it?

Reply:
sure, which one path can return full list ?

Reply:
/api/locales should return locale keys list
Caj be checked in swagger

Reply:
Fixed, please check